meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -249,6 +253,31 @@ function version_gt() {
     test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1"
 }
 
+function cachedpull() {
+    local _image="$1"
+    local _image_escaped=$(echo "$_image" | sed -r "s/[\/\.:-]+/_/g")
```

Comment:
Translating all those was just an aesthetics choice. Replacing as well was intentional as well but on a second thought I don't see the obvious benefit in it so I will remove it.